solutions/standard/variables.tf

```diff
@@ -118,13 +124,13 @@ variable "kms_endpoint_type" {
 variable "kms_key_ring_name" {
   type        = string
   default     = "sm-cos-key-ring"
-  description = "The name to give to the new KMS key ring that will be used to store the KMS key to enable advanced, customer-managed encryption for your Secrets Manager secrets. Only required if not supplying an existing KMS root key CRN."
+  description = "The name to give the Key Ring which will be created for the Secrets Manager COS bucket Key. Not used if supplying an existing Key. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
 }
 
 variable "kms_key_name" {
   type        = string
   default     = "sm-cos-key"
-  description = "The name to give to the new KMS root key that will be used to enable advanced, customer-managed encryption for your Secrets Manager secrets. Only required if not supplying an existing KMS root key CRN."
+  description = "The name to give the Key which will be created for the Secrets Manager COS bucket. Not used if supplying an existing Key. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
```

Comment:
why did you change the description to say COS bucket? There is no COS bucket here

Reply:
As can be seen from the main [variable.tf](https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/blob/60accca4ed5a83b4afd45f002ec831ae711be226/solutions/standard/variables.tf#L127) the description had COS bucket in it.

Reply:
The earlier variable description had COS bucket in it but it was modified in this [PR](https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/commit/4593f02bf30c3b59b820a650b360bff813bacbad) which may not have been taken care while merging the main branch.